src/main/scala/tech/cryptonomic/conseil/config/package.scala

```diff
@@ -0,0 +1,15 @@
+package tech.cryptonomic.conseil
+
+import com.typesafe.scalalogging.LazyLogging
+
+package object config extends LazyLogging {
```

Comment:
Is the package object config (all in lower case) some special type of object in Scala?

Reply:
indeed, it's a special singleton instance whose members are visible to all the code under the same package.

e.g. every piece of code in the `tech.cryptonomic.conseil.config` package will have visibility of the `printConfigurationError` method.

You can import the members of a package object in scope by simply importing the package itself
i.e. `import tech.cryptonomic.conseil.config._`